Maquetado/scss/grid.scss

```diff
@@ -0,0 +1,99 @@
+.tech-container-grid {
+  display: grid;
+  grid-template-columns: 1fr 1 fr 1fr;
+  grid-template-rows: auto;
+}
+
+.container-wrapper {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  margin-bottom: 70px;
+  padding: 0 70px;
+}
+
+.tech1 { 
+  border-right: solid 2px $green;
```

Comment:
I'd move non grid related styles to a `big-tech` class that isolates the styles related to the big tech.